### ISSUE TITLE
BaM is taking a very long time to load on multiple platform

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/build-a-molecule#224]
@mattpen found that on his Android and Chromebook devices, BaM takes quite a while to load. Android phone ~20 seconds. Chromebook ~10 seconds.
@KatieWoe can you describe the devices you have tested BaM and note their load performance:

Device and OS Version | Load time
------------- | ------------

**Comment by KatieWoe** [comment]
Device and OS Version | Load time
------------- | ------------
Win 10 Chrome | Load time 1
Win 10 Firefox | Load time 2
Mac 11 Safari | Load time 3
iPad 14 | Load time 4
ChromeOS | Load time 5
Android Tablet | Load time 6

**Comment by mattpen** [comment]
Assigning to @jonathanolson since he is the responsible dev for build-a-molecule.

**Comment by jonathanolson** [comment]
This is partially expected, it has to load a LOT of data. @oliver-phet, any preferences on the balance between load-time of the sim and download size? At a certain level there's a tradeoff, but I could take a look into optimizing things (I'm sure I could help improve things).

**Comment by oliver-phet** [comment]
@jonathanolson do you have a perspective on why the sim takes _so much longer_ to load on Android and Chrome specifically? When I load the sim on my iPhone (in our iOS app or mobile safari) the sim only takes ~3 seconds to load. I appreciate the tradeoff between load-time and download size, but it is surprising that those two platforms are experiencing such long load times.

**Comment by jonathanolson** [comment]
> do you have a perspective on why the sim takes so much longer to load on Android and Chrome specifically?

I'm not sure. It's a very specific CPU task that allocates a lot of memory. It's probably fairly dependent on the CPU and memory, and heavily dependent on browser optimizations and GC.

**Comment by jonathanolson** [comment]
I could profile it presumably to see why this might be happening?

**Comment by oliver-phet** [comment]
Yes, @jonathanolson profiling sounds like the next best step! This would be good to diagnose before our next Chromebook app goes out.
